Packs/TruSTAR/Integrations/TruSTAR_V2/README.md

```diff
@@ -0,0 +1,574 @@
+Use the TruSTAR v2 integration to manage reports, indicators and phishing submissions.
```

Comment:
The README seem to be copy and pasted from the previous integration, and it is not according to our standard
please regenerate as described in https://xsoar.pan.dev/docs/integrations/integration-docs